example/my-app/src/Chat.tsx

```
@@ -41,81 +44,119 @@ function mapChatResultToMessages(result: ChatResult): Message[] {
+    }
+
+    switch (resultType) {
         case "main.SystemResponse":
             return result["main.SystemResponse"].ToolCalls || [];
 
+        case "main.UserResponse":
+            // UserResponse doesn't have tool calls
+            return [];
+
         case "main.ChatResponses":
             const responses = result["main.ChatResponses"]["Responses"] || [];
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)